pkg/api/secretbootstrap/secretboostrap_test.go

```diff
@@ -209,6 +217,41 @@ func TestLoadConfigFromFile(t *testing.T) {
 	}
 }
 
+func TestRoundtripConfig(t *testing.T) {
+	testCases := []struct {
+		name string
+	}{
+		{
+			name: "basic base",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			c := &Config{}
+			inFile := filepath.Join("testdata", fmt.Sprintf("%s.yaml", t.Name()))
+			err := LoadConfigFromFile(inFile, c)
+			if err != nil {
+				t.Errorf("error loading config file: %v", err)
+			}
+
+			outFile := filepath.Join("testdata", fmt.Sprintf("%s_out.yaml", t.Name()))
+			err = SaveConfigToFile(outFile, c)
+			if err != nil {
+				t.Errorf("error saving config file: %v", err)
+			}
+
+			in, _ := ioutil.ReadFile(inFile)
```

Comment:
`t.Fatal()` on these errors :)

Reply:
This didn't get addressed

Reply:
Looking into what happened here, I have it changed to `Fatal` locally...